ReactiveCocoaTests/Swift/SignalLifetimeSpec.swift

```diff
@@ -0,0 +1,224 @@
+//
+//  SignalLifetimeSpec.swift
+//  ReactiveCocoa
+//
+//  Created by Vadim Yelagin on 2015-12-13.
+//  Copyright (c) 2015 GitHub. All rights reserved.
+//
+
+import Result
+import Nimble
+import Quick
+import ReactiveCocoa
+
+class SignalLifetimeSpec: QuickSpec {
+	override func spec() {
+		describe("init") {
```

Comment:
I don't know. There's `describe("init")` in SignalSpec and SignalProducerSpec too.